darwin/cli.py

```diff
@@ -45,7 +45,7 @@ def main() -> None:
     except requests.exceptions.ConnectionError:
         f._error("Darwin seems unreachable, please try again in a minute or contact support.")
     except GracefulExit as e:
-        f.error(e.message)
+        f._error(e.message)
```

Comment:
Not sure how this made it in but may as well stay

Reply:
Yeah, that is a bit weird, but nvm as you say.